### ISSUE TITLE
machine_image not supported?

### ISSUE THREAD

**Issue by johnsmyth** [CenturyLinkCloud/chef-provisioning-vsphere#25]
I tried to use machine_image, but received the error:
ChefProvisioningVsphere::VsphereDriver does not implement create_image

Any plans for implementing this, or some other resource to build vmware templates with?

**Comment by mwrock** [comment]
Right, this driver does not currently support `machine_image`. I don't have any plans to add it but would gladly accept PRs toward this effort.

**Comment by johnsmyth** [comment]
cool - working on it...

**Comment by mschnell42** [comment]
johnsmyth  Have you started any machine_image work?  Could you use a guinea pig for anything you have in hand.?

**Comment by johnsmyth** [comment]
This was de-prioritized before I could finish it, and I don't have access to the resources that I would need to complete it any more. My partially completed code is pushed to my fork (https://github.com/johnsmyth/chef-provisioning-vsphere) if you want to use it as a base though.

**Comment by mschnell42** [comment]
Bummer. While, at this point, I'm happy to play guinea pig, I'm afraid that I am too new to this Chef & Ruby (and the vCenter APIs) to be able to make any speedy progress in this area, given my other activities. Give me another month or two and I might be ready to tackle it, but right now I'd just get lost. :-)
